005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -192,6 +192,39 @@ \section{Results}
         Utility                 & 0.009                 & 0.013                  & 0.034                 \\ \hline
     \end{tabular}
     \caption{Experimental results.}
+    \label{tab:InitalResults}
 \end{table}
-These experiments were run for 500 days, with 50 food for 10 agents. This means there was enough food for every agent to survive at critical level but not enough to satisfy all agents. For this reasons, agents need to communicate and self organise in order to decrease the amount of deaths in this strenuous circumstances.\par 
-The results indicate that the agents are capable of reducing their deaths significantly by following the research on group identity. As expected from this research,  classical game theory methods of selfishness are outperformed by more intricate methods of organisation in a collective, such as the one exposed by group identity.\par
+These experiments were run for 500 days, with 50 food for 10 agents and a reshuffle period of 7 days. Each experiment was run 3 times and the average taken. This means there was enough food for every agent to survive at critical level but not enough to keep them all out of critical condition. For this reasons, agents need to communicate and self organise in order to decrease the amount of deaths in this strenuous circumstances.
+
+As you can see in \ref{tab:InitalResults}, the results indicate that the agents are capable of  significantly reducing their deaths compared to random agents. This difference arises due to the Team 3 agents’ increased communication capabilities via treaties and messages, allowing them to assume a group identity. Random agents take an approach which more closely embodies the selfish approach dictated by classical game theory. As is expected, this is outperformed by a higher degree of organisation in a collective, which successfully models group identity and collective human survival instinct.
+
+\ref{fig:AvgDeaths} further shows how, even as the food in the platform changes, team 3’s agent always outperforms the selfish and random agents.
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.6\linewidth]{005_team_3_agent_design/images/Average_Deaths_of_Agents_Against_Total_Amount_of_Food_On_Platform.jpg}
+    \caption{Average deaths of agents against total amount of food on platform.}
+    \label{fig:AvgDeaths}
+\end{figure}
+
+In addition, we ran tests where our agent was placed into the tower along with another teams’ agent. Five agents from team 3 and five from the respective team in \ref{fig:AgainstOtherTeams} were simulated for 500 days and 100 total food on the platform. In \ref{fig:AgainstOtherTeams} you can see the cumulative deaths of both agent types.  
```

Comment:
numbers instead of spelling them out?
```suggestion
In addition, we ran tests where our agent was placed into the tower along with another teams’ agent. 5 agents from team 3 and 5 from the respective team in \ref{fig:AgainstOtherTeams} were simulated for 500 days and 100 total food on the platform. In \ref{fig:AgainstOtherTeams} you can see the cumulative deaths of both agent types.  
```

Reply:
I personally prefer the other one, to differentiate Team 5 and others